chain/power_table_view_test.go

```diff
@@ -25,40 +25,44 @@ func TestTotal(t *testing.T) {
 
 	ctx := context.Background()
 
-	power := uint64(19)
-	bs, _, st := requireMinerWithPower(ctx, t, power)
+	numCommittedSectors := uint64(19)
+	bs, _, st := requireMinerWithPower(ctx, t, numCommittedSectors)
 
 	actual, err := (&consensus.MarketView{}).Total(ctx, st, bs)
 	require.NoError(t, err)
 
-	assert.Equal(t, power, actual)
+	expected := types.OneKiBSectorSize.Mul(types.NewBytesAmount(numCommittedSectors))
+
+	assert.Equal(t, expected.Uint64(), actual.Uint64())
 }
 
 func TestMiner(t *testing.T) {
 	tf.UnitTest(t)
 
 	ctx := context.Background()
 
-	power := uint64(12)
-	bs, addr, st := requireMinerWithPower(ctx, t, power)
+	numCommittedSectors := uint64(12)
+	bs, addr, st := requireMinerWithPower(ctx, t, numCommittedSectors)
 
 	actual, err := (&consensus.MarketView{}).Miner(ctx, st, bs, addr)
 	require.NoError(t, err)
 
-	assert.Equal(t, power, actual)
+	expected := types.OneKiBSectorSize.Mul(types.NewBytesAmount(numCommittedSectors))
+
+	assert.Equal(t, expected.Uint64(), actual.Uint64())
 }
 
-func requireMinerWithPower(ctx context.Context, t *testing.T, power uint64) (bstore.Blockstore, address.Address, state.Tree) {
+func requireMinerWithPower(ctx context.Context, t *testing.T, numCommittedSectors uint64) (bstore.Blockstore, address.Address, state.Tree) {
```

Comment:
This function name is now a bit misleading about the uint value.

Reply:
Indeed. Fixed.